lib/mjml/parser.rb

```diff
@@ -36,10 +36,10 @@ def render
     #
     # @return [String] The result as string
     # rubocop:disable Style/OptionalBooleanParameter: Fixing this offense would imply a change in the public API.
-    def run(in_tmp_file, beautify = true, minify = false, validation_level = 'strict')
+    def run(in_tmp_file, beautify = true, minify = false, validation_level = 'strict', fonts = Mjml.fonts)
```

Comment:
here I wasn't sure what was the best way to pass defaults fonts agrument (as it's a long hash). 
Is it possible to directly use Mjml module defined options ?

Reply:
@clairezed We could try getting the module options directly or else look at not calling `--config.fonts` if it hasn't been set?